cli/src/main/java/com/liferay/blade/cli/StringPrintStream.java

```diff
@@ -48,7 +49,27 @@ public static StringPrintStream newInstance(Charset charset) {
 
 	@Override
 	public String get() {
-		return new String(_outputStream.toByteArray(), _charset);
+		StringBuilder stringBuilder = new StringBuilder();
```

Comment:
Can you create a subclass called 'FilteringPrintStream' that will override the get and perform the filters based on regexes instead?